doc/about.rst

```diff
@@ -85,8 +85,8 @@ citations to the following paper:
 If you want to cite scikit-learn for its API or design, you may also want to consider the
 following paper:
 
-  `API design for machine learning software: experiences from the scikit-learn
-  project <https://arxiv.org/abs/1309.0238>`_, Buitinck *et al.*, 2013.
+  API design for machine learning software: experiences from the scikit-learn
+  project, Buitinck *et al.*, 2013. :arxiv:`1309.0238`
```

Comment:
I just realize that it is possible to give an explicit caption https://github.com/scikit-learn/scikit-learn/blob/fa7327b94ca9a3d87850ff4b8ed953dcfda9f6cd/doc/sphinxext/doi_role.py#L10-L11
Do you mind checking this solution that would be more consistent with other references?
```suggestion
  :arxiv:`API design for machine learning software: experiences from the scikit-learn
  project <1309.0238>`, Buitinck *et al.*, 2013.
```
Thanks!